docs/src/using-criticalup/creating-sample-manifest.rst

```diff
@@ -0,0 +1,33 @@
+.. SPDX-FileCopyrightText: The Ferrocene Developers
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+
+.. _creating_sample_manifest:
+
+Creating sample manifest
+========================
+
+This chapter describes a way to quickly create a sample ``criticalup.toml`` for your project.
+
+Creating a sample manifest file
+-------------------------------
+
+Running the ``init`` command will create a file ``criticalup.toml`` in the current working directory.
```

Comment:
So https://criticalup.ferrocene.dev/using-criticalup/toolchain-management.html#toolchain-management already discusses manifests a bit, and it assumes you've created one.

While this change does *add* documentation, I'm not sure it builds on the existing documentation and it feels very much like:

```
/// An iterator over a number of T
struct Iterator<T> {}
```

Perhaps instead we could rework the "Toolchain Management" page to mention this init command to create that manifest and detail how to alter it to be useful.

Reply:
Thank you. I made the following changes 

- Added this section of creating sample manifest to Toolchain management page. (I had already included this section in References >> criticalup.toml)
- Reordered the pages, making "creating sample manifest" next to Toolchain management.
- Replaced all release versions from 24.05.0 to 25.02.0 since thats the latest. That allows is to remove a warning in the Toolchain management of 24.05 being Linux only.
- Used sample manifest output in the docs for consistency.